app/renderer/css/main.css

```diff
@@ -170,6 +170,35 @@ body {
     border-radius: 4px;
 }
 
+.tab .server-icons:not(.alt-icon):before { 
+    content: attr(data-content);
+    font-family: Verdana;
+    font-weight: 600;
+    font-size: 22px;
+    border: 2px solid #222c31;
+    position: absolute;
+    left: 0;
+    width: 35px;
+    margin-left: 17%;
+    border-radius: 4px;
+}
+    content: attr(data-content);
```

Comment:
@NikhilPhalange  You forgot to remove the parts of your code that was replaced.

Reply:
@abhigyank Thanks a lot, I was incorporating the suggested changes for the first time, never did it before. Didn't know how it worked. Please inform me in case anything is left.